docs/index.md

```diff
@@ -152,15 +155,15 @@ We enforce API rate limits in two ways:
 
 ### Daily Requests
 
-Pardot Pro customers are allocated 25,000 API requests per day. Pardot Ultimate customers can make up to 100,000 API requests a day. 
-These limits reset at the beginning of the day based on your accounts time zone settings. Any request made exceeding the 
+Pardot Pro customers are allocated 25,000 API requests per day. Pardot Ultimate customers can make up to 100,000 API requests a day.
+These limits reset at the beginning of the day based on your accounts time zone settings. Any request made exceeding the
 limits will result in an [error code 122](/kb/error-codes-messages/#error-code-122)
 
 You can check you current daily usages in the accounts "usage and limits" page.
 
 ### Concurrent Requests
 
-In order to interact with our API more efficiently, you can have up to five concurrent API requests. Any connection over five 
+In order to interact with our API more efficiently, you can have up to five concurrent API requests. Any connection over five
 will result in an [error code 66](/kb/error-codes-messages/#error-code-66) response.
 
 ## Sample Code
```

Comment:
We will also need to update the sample code below. Below is an updated sample code. GitHub doesn't allow for comments on code outside the changed code so I'm unable to suggest directly.

```php
<?php

/**
 * Call the Pardot API and get the raw XML response back
 *
 * @param string $url the full Pardot API URL to call, e.g. "https://pi.pardot.com/api/prospect/version/3/do/query"
 * @param array $data the data to send to the API
 * @param string $method the HTTP method, one of "GET", "POST", "DELETE"
 * @param array $headers array of headers to send to the API
 * @return string the raw XML response from the Pardot API
 * @throws Exception if we were unable to contact the Pardot API or something went wrong
 */
function callPardotApi($url, $data, $method = 'GET', $headers = null)
{
    // build out the full url, with the query string attached.
    $queryString = http_build_query($data, null, '&');
    if (strpos($url, '?') !== false) {
        $url = $url . '&' . $queryString;
    } else {
        $url = $url . '?' . $queryString;
    }

    $curl_handle = curl_init($url);

    // wait 5 seconds to connect to the Pardot API, and 30
    // total seconds for everything to complete
    curl_setopt($curl_handle, CURLOPT_CONNECTTIMEOUT, 5);
    curl_setopt($curl_handle, CURLOPT_TIMEOUT, 30);

    // https only, please!
    curl_setopt($curl_handle, CURLOPT_PROTOCOLS, CURLPROTO_HTTPS);

    // ALWAYS verify SSL - this should NEVER be changed. 2 = strict verify
    curl_setopt($curl_handle, CURLOPT_SSL_VERIFYHOST, 2);

    // return the result from the server as the return value of curl_exec instead of echoing it
    curl_setopt($curl_handle, CURLOPT_RETURNTRANSFER, 1);

    if (strcasecmp($method, 'POST') === 0) {
        curl_setopt($curl_handle, CURLOPT_POST, true);
    } elseif (strcasecmp($method, 'GET') !== 0) {
        // perhaps a DELETE?
        curl_setopt($curl_handle, CURLOPT_CUSTOMREQUEST, strtoupper($method));
    }

    // add any headers that were specified
    if ($headers) {
        curl_setopt($curl_handle, CURLOPT_HTTPHEADER, $headers);
    }

    $pardotApiResponse = curl_exec($curl_handle);
    if ($pardotApiResponse === false) {
        // failure - a timeout or other problem. depending on how you want to handle failures,
        // you may want to modify this code. Some folks might throw an exception here. Some might
        // log the error. May you want to return a value that signifies an error. The choice is yours!

        // let's see what went wrong -- first look at curl
        $humanReadableError = curl_error($curl_handle);

        // you can also get the HTTP response code
        $httpResponseCode = curl_getinfo($curl_handle, CURLINFO_HTTP_CODE);

        // make sure to close your handle before you bug out!
        curl_close($curl_handle);

        throw new Exception("Unable to successfully complete Pardot API call to $url -- curl error: \"".
                                "$humanReadableError\", HTTP response code was: $httpResponseCode");
    }

    // make sure to close your handle before you bug out!
    curl_close($curl_handle);

    return $pardotApiResponse;
}

//available from https://pi.pardot.com/account
$userKey = '12345678890abcdef12345678890abcdef'; 

//this will log in and retrieve your API Key (good for 1 hour)
$loginResponse = callPardotApi('https://pi.pardot.com/api/login/version/3',
    array(
        'email' => 'your.email@example.com',
        'password' => 'password1234',
        'user_key' => $userKey,
        'format' => 'json'
    ),
    'POST'
);
$apiKey = json_decode($loginResponse, true)['api_key'];

echo callPardotApi('https://pi.pardot.com/api/prospect/version/3/do/query',
    array(
        'limit' => '1',
        'format' => 'json'
    ),
    'GET',
    array(
        //create the Authorization header from the user_key and api_key
        "Authorization: Pardot user_key=$userKey,api_key=$apiKey"
    )
);
```